main.tf

```diff
@@ -108,3 +129,14 @@ resource "azurerm_template_deployment" "storage_account_data_protection" {
         }
     DEPLOY
 }
+
+
+module "storage_account_containers" {
```

Comment:
probably doesn't need to be a module? could we move this back to this folder level but keep in a separate file?

Reply:
I have moved this to how I believe you have requested it